### PR TITLE
fixed test to run on mac os x (PHP_OS is set to „Darwin“ there)

### PR DESCRIPTION
**Pull request by empi89**
My Mac OS X tried to exec a windows cmd because PHP_OS is set to "Darwin". I have added a method to check if the system is windows and replaced the strpos-code with it.

### CODE REVIEW

**Comment by ghost** [comment]
Sorry to have done this, but I was in the middle of working on Box 3.0.

Could you please re-do your changes on the 2.0 branch?

I had also created a new [contribution guide](https://github.com/kherge/Box/blob/master/CONTRIBUTING.md), so if you could read it too I would really appreciate it!

File: src/tests/KevinGH/Box/Tests/ConfigurationTest.php [hunk 1]
```diff
@@ -1219,4 +1219,13 @@ private function setConfig(array $config)
     {
         $this->setPropertyValue($this->config, 'raw', (object) $config);
     }
+    
+    private function isWindows()
+    {
+        if (false === strpos(strtolower(PHP_OS), 'darwin') && $this->isWindows() ) {
```
Comment (ghost) [inline src/tests/KevinGH/Box/Tests/ConfigurationTest.php:1225]: On non-darwin machines, wouldn't this cause an infinite recursion?

Comment (empi89) [inline src/tests/KevinGH/Box/Tests/ConfigurationTest.php:1225]: Ahh you are correct! That is a mistake with search and replace, I'm really sorry and I'll fix this!

**Comment by empi89** [comment]
I close this PR in favor of https://github.com/kherge/Box/pull/61.

**Pull request closed without merging**